crates/loans/src/types.rs

```diff
@@ -1,8 +1,25 @@
-use crate::InterestRateModel;
+use crate::{Config, InterestRateModel};
+use currency::Amount;
 use frame_support::pallet_prelude::*;
-use primitives::{CurrencyId, Rate, Ratio};
+use primitives::{CurrencyId, Liquidity, Rate, Ratio, Shortfall};
 use scale_info::TypeInfo;
 
+/// Container for account liquidity information
+#[derive(Encode, Decode, Eq, PartialEq, Clone, RuntimeDebug, TypeInfo)]
+pub struct AccountLiquidity<T: Config> {
+    pub liquidity: Amount<T>,
+    pub shortfall: Amount<T>,
+}
```

Comment:
It seems to me that this could more elegantly be described like this?

```suggestion
pub enum AccountLiquidity<T: Config> {
    
    pub liquidity: Amount<T>,
    pub shortfall: Amount<T>,
}
```